### PR TITLE
Improve TCB index page

### PR DESCRIPTION
**Pull request by mosabua**
- Reword intro and improve layout
- Create section for upcoming episodes

### CODE REVIEW

File: broadcast/index.md [hunk 1]
```diff
@@ -8,21 +8,46 @@ show_hero: true
 <div class="row spacer-60">
 <div markdown="1" class="col-lg-8">
 
-## About
+Trino Community Broadcast is a show about the Trino project and community.
```
Comment (mosabua) [inline broadcast/index.md:11]: reworded this

File: broadcast/index.md [hunk 2]
```diff
@@ -8,21 +8,46 @@ show_hero: true
...
+Topics include the data sources, clients, add-ons, platforms and usages of
+Trino. We interview users, contributors, partners, and other community members
+about their connection to Trino, look at demos together, and chat about
+the tech stacks, the code, and often explain concepts along the way.
+
+Each episode also brings you news about the latest releases, events, and
+interesting developments in the ecosystem around Trino.
+
+## Upcoming episodes
```
Comment (mosabua) [inline broadcast/index.md:20]: Added this

File: broadcast/index.md [hunk 3]
```diff
@@ -8,21 +8,46 @@ show_hero: true
...
+<dt>4 Apr 2024: Trino Community Broadcast 58 - Mitzu</dt>
+<dd>
+István Mészáros chats about his company and analytics tool
+<a href="../ecosystem/client.html#mitzu">Mitzu</a> - a friendly, no-code product
+analytics tool for warehouse-first companies that supports Trino.</dd>
+</dl>
+
+Find [more details and other events on our calendar]({{site.url}}/community.html#events).
+
+Trino users, contributors, and partners, we want to hear from you! If you have a
```
Comment (mosabua) [inline broadcast/index.md:45]: Reworded and moved down into context

File: broadcast/index.md (hunk 1 above)
Comment (colebow) [inline broadcast/index.md:11]:
```suggestion
The Trino Community Broadcast is a show about the Trino project and community.
```

**Review by colebow — APPROVED** [review]
One nit but otherwise LGTM

**Pull request merged**